src/utils.js

```diff
@@ -7,7 +7,7 @@
  *  The URL host, required.
  * @param {string} port
  *  The URL port, default to empty string.
- * @param {string} [type]
+ * @param {any} [type]
```

Comment:
Why changing string to any?
It was more clear before

Reply:
It can be `false` as well as a `number`

Reply:
I don't know about the doc, but in ts you can write:
`string | number | false`
It is still more precise than `any`